test/suites/clustering.sh

```diff
@@ -732,6 +737,13 @@ test_clustering_network() {
   ! LXD_DIR="${LXD_ONE_DIR}" lxc network create "${net}" --target node2 || false
   LXD_DIR="${LXD_ONE_DIR}" lxc network show "${net}" | grep status: | grep -q Pending
 
+  # A container can't be associated with a pending network.
```

Comment:
@freeekanayaka perhaps could be clarified with "container can't be started when associated with a pending network"?

Reply:
Makes sense, fixed.